client/webserver/live_test.go

```diff
@@ -283,10 +283,17 @@ var tExchanges = map[string]*core.Exchange{
 			mkid(3, 22): mkMrkt("doge", "mona"),
 		},
 		Connected: true,
-		Fee: &core.FeeAsset{
-			ID:    42,
-			Confs: 1,
-			Amt:   1e8,
+		RegFees: map[string]*core.FeeAsset{
+			"dcr": &core.FeeAsset{
+				ID:    42,
+				Confs: 1,
+				Amt:   1e8,
+			},
+			"btc": &core.FeeAsset{
+				ID:    0,
+				Confs: 1,
+				Amt:   1e6,
+			},
```

Comment:
Only had to update these because the running against the test server in the legacy mode was resulting in an error at 

https://github.com/decred/dcrdex/blob/9e1e0cb8787ca6ec48463ac006084c3a1e0ee461/client/webserver/site/src/js/forms.js#L334

Are we already done supporting older servers?